src/main/java/org/jenkinsci/plugins/badge/BadgeAction.java

```diff
@@ -5,36 +5,67 @@
 import jenkins.model.Jenkins;
 
 import org.kohsuke.stapler.HttpResponse;
-import org.kohsuke.stapler.QueryParameter;
-
 
 /**
-* @author Kohsuke Kawaguchi
-*/
+ * @author Kohsuke Kawaguchi
+ */
 public class BadgeAction implements Action {
+
+    /**
+     * TO DO
+     */
     private final BadgeActionFactory factory;
 
+    /**
```

Comment:
[Checkstyle] WARNING: First sentence of Javadoc is incomplete (period is missing) or not present.